volume/store/store.go

```diff
@@ -23,14 +23,24 @@ const (
 )
 
 type volumeMetadata struct {
-	Name   string
-	Labels map[string]string
+	Name    string
+	Labels  map[string]string
+	Options map[string]string
 }
 
 type volumeWrapper struct {
 	volume.Volume
-	labels map[string]string
-	scope  string
+	labels  map[string]string
+	scope   string
+	options map[string]string
+}
+
+func (v volumeWrapper) Options() map[string]string {
+	options := map[string]string{}
+	for key, value := range v.options {
+		options[key] = value
+	}
+	return options
```

Comment:
It would probably make sense for labels as well, but that's out of scope for this PR